configs/dev-genesis.js

```diff
@@ -52,7 +52,8 @@ config.mail.connection = {
 };
 config.mail.send = false;
 config.mail.vars = {
-  productionMode: config.views.vars.productionMode,
+  // FIXME: what is this?
+  productionMode: '',
```

Comment:
IIRC, which I probably don't, it's the thing that determines whether to contact the SMTP server and fire off an email or not. In non-production mode, it doesn't sent email, but logs what would have been sent to the console.

Reply:
Add an item to roadmap to deal with email configuration.